zerver/openapi/zulip.yaml

```diff
@@ -1048,6 +1048,59 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/JsonSuccess'
+  /user_groups:
+    get:
+      description: Get all user groups of the realm.
+      security:
+      - basicAuth: []
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/JsonSuccess'
+                - properties:
+                    user_groups:
+                      type: array
+                      items:
+                        type: object
+                      description: A list of `user_group` objects, which contain a
+                        `description`, a `name`, their `id` and the list of members
+                        of the user group.
+                - example:
+                    {
```

Comment:
Even though the YAML file is indented with 2 spaces, we use 4 spaces for JSON responses to make them more readable.

Reply:
@YagoGG , I added spaces for responses and removed `400:` fixture.